app/Youtube/Video.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Youtube;
+
+/**
+ * @author Michael Phillips <michael.phillips@realpage.com>
+ */
+class Video
```

Comment:
Suggestion: I would name this class more descriptively as something like `VideoUrl`.